free-3.md

```diff
@@ -4,40 +4,62 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Super Smash Brothers Melee (SSBM) is an extremely popular competitive fighting game (there were about 200,000 concurrent viewers for last years biggest tournament).  The game is very deep, and top players engage in a lot of "theory-crafting" (theorizing about hypothetical situations that could come up in the game).  Because there is no systematic way of notating scenarios in SSBM, theory-crafting discussions over the internet can be quite difficult to maintain.
+My language would allow users to describe the details surrounding the hypothetical in-game scenario and it would decide who the "winner" (the opponent who is in an advantageous position at the end of the scenario) is based on the game's frame data.
```

Comment:
This is awesome, one of my ideas is essentially the same thing but for DOTA 2. Great minds think alike :)